pallets/time-release/src/mock.rs

```diff
@@ -125,6 +125,7 @@ pub struct ExtBuilder;
 impl ExtBuilder {
 	pub fn build() -> sp_io::TestExternalities {
 		let mut t = frame_system::GenesisConfig::default().build_storage::<Runtime>().unwrap();
+		MockBlockNumberProvider::set(0);
```

Comment:
Tests can run in parallel so having a static value being set and read from different tests without any proper locking seems to interfere with having isolated tests.

Reply:
This is called before every test is run.

Reply:
I see. I had not thought about it. Hmmm... I think we do this in other places where we update constants.

Reply:
ahh ok. I think we are ok because we are using parameter types and we reset storage on every run.